packages/tables/resources/views/components/pagination/records-per-page-selector.blade.php

```diff
@@ -2,13 +2,13 @@
     'options',
 ])
 
-<div class="filament-tables-pagination-records-per-page-selector flex items-center space-x-2 rtl:space-x-reverse ">
+<div class="flex items-center space-x-2 filament-tables-pagination-records-per-page-selector rtl:space-x-reverse ">
```

Comment:
```suggestion
<div class="flex items-center space-x-2 filament-tables-pagination-records-per-page-selector rtl:space-x-reverse">
```

Reply:
And make sure to move the `filament-` class to the start!